code/modules/mob/language/station.dm

```diff
@@ -128,12 +128,16 @@
 	exclaim_verb = "sings"
 	whisper_verb = "hums"
 	colour = "skrellfar"
-	key = "p"
-	space_chance = 30
-	flags = WHITELISTED
-	syllables = list("qr","qrr","xuq","qil","quum","xuqm","vol","xrim","zaoo","qu-uu","qix","qoo","zix", "...", "oo", "q", "nq", "x", "xq", "ll", "...", "...", "...") //should sound like there's holes in it
-
-
+ 	key = "p"
```

Comment:
Lines 131-135 have a leading space, which needs to be removed.
Lines 137-139 are space-indented rather than tab-indented (Thanks discord), and they need to be the latter for byond